Sources/TurnstileWeb/Protocols/OAuth/OAuthParameters.swift

```diff
@@ -72,8 +72,12 @@ public class OAuthParameters {
     public var header: String {
         let partialHeader = parameterDictionary.sorted { $0.0.key < $0.1.key }
         .map { (keyValuePair) -> String in
-            let values = keyValuePair.value.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlQueryAllowed) ?? ""
```

Comment:
So the `.urlQueryAllowed` had the right characters from Twitters docs but it wasn't encoding characters such as `+` & `=` because those are allowed in query parameters eg. `?name=kaden&email=test@example.com` and so when these characters were part of the oauth params they didn't get their percent encoding and the API rejected it.